meetings/2025-05-14.md

```diff
@@ -0,0 +1,348 @@
+
+elf Pavlik: 22:46 
+ Okay, that's nice. My main question would be because I didn't want to like throw even more use cases knowing that there's already a lot of it. But I would still like to work with some kind of variants of use cases. So I give one example and again it's based on some kind of requirements that we found when we were implementing SAI that for example when we talk about sharing data. I think the current use case is mentioning like authoring tool which I understand implies application but we had at some point requirement that we want to be able to share data but do not disclose to the specific application the whole social graph. So of course, if in the data that the application has access to, there are some web IDs, the web, the application will kind of, you know, have access just to, to those web IDs in the data. But we do not want to like just completely give, give a wild card access to the whole social graph of application. So that's one of the reasons that for example, the, the main, main flow implemented at this moment is that the application redirects to the authorization agent, authorizations happens over there and then user goes back to application. So this way users can share data without disclosing the social graph to application. And my question is, like, where would this kind of requirement and use case belong? Would that be like a variant of an existing use case? Like how do you see covering those like, you know, variants of how things can be done and what requirements are kind of related to, you know, those little different conditions?
+
+Christoph Braun (KIT)  uvdsl: 24:31 
+ I'm not certain where they're gonna go right now. I'm still, like I said, I just began looking through the existing use cases as they are to try to get my head around them all. My personal preference is I'd rather hear any and all use cases, even if they're new ones, even if they aren't part of that current pool. It's considering everything that is of interest to make sure that we have the foundations for something that's extensible, that doesn't require a rewrite.
+
+elf Pavlik: 25:01 
+ Go ahead, Chris. Go ahead, Christoph.
+
+Christoph Braun (KIT)  uvdsl: 25:04 
+ Yeah, I agree with that. And I, so I'd agree with that. And it makes the subtleties apparent or more obvious because I think some people from the community, they're, they're quite clear on that we need this distinction between the authorization app or agent or whatever mechanism there is to, to authorize access, that this should be distinct from some consuming business application or personal application. Right. So there, there is this notion that people are aware of and I believe these people, me included, will like fight for that. But still, if there are any use cases you can give us as ammunition, so to speak, that would be great.
```

Comment:
```suggestion
 Yeah, I agree with that. And it makes the subtleties apparent or more obvious because I think some people from the community, they're quite clear on that we need this distinction between the authorization app or agent or whatever mechanism there is to authorize access, that this should be distinct from some consuming business application or personal application. Right. So there, there is this notion that people are aware of and I believe these people, me included, will like fight for that. But still, if there are any use cases you can give us as ammunition, so to speak, that would be great.
```